front/lib/models/plan.ts

```diff
@@ -105,6 +106,11 @@ Plan.init(
       type: DataTypes.INTEGER,
       allowNull: false,
     },
+    maxSizeMbStaticDataSources: {
```

Comment:
No Static :) It is enforced for all data sources

Reply:
Yes today it is. I can rename to match the names for the current limitations PlanType, but we will have to rework and rename this again when we properly handle the limitations that we want in the pricing (the pricing discussion was still in progress when I implementing this). I don't think it's more urgent than working on plugging Stripe tho, but I'll do it 👍🏻

Reply:
Very well possible that this will be a "new" limitation also

Reply:
Yes of course, my point was only about priorities and pricing being more urgent 🙇🏻‍♀️ 
I added a commit to match exactly the names of the type, good for you? 🙏🏻

Reply:
yes yes!